src/components/TabBar/TabBar.css

```diff
@@ -41,11 +41,4 @@
     border-color: var(--primaryLinkColor);
     color: color(var(--colorMuted) lightness(85%));
   }
-
-  @media only screen and (max-width: 1439px) {
-    white-space: nowrap;
```

Comment:
Change this media query to be smaller,  add media queries to the overflow code. This solution won't work for mobile users.